src/codeu/chat/client/ClientUser.java

```diff
@@ -109,6 +114,18 @@ public void showAllUsers() {
     }
   }
 
+  public User searchByName(String s) {
+    updateUsers();
```

Comment:
Might be worth putting a comment to clarify why an update call is required here. It's not obvious to me why usersByName.all() wouldn't have fresh data. Probably want a new line under this too.